src/schema/schema_create.c

```diff
@@ -278,7 +280,17 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
             /* Try to recreate the associated metadata from the imported data source. */
             WT_ERR(__wt_import_repair(session, uri, &fileconf));
         }
-        WT_ERR(__wt_metadata_insert(session, uri, fileconf));
+
+        /*
+         * Strip the tiered storage shared configuration from the metadata as it is not required to
+         * be persisted.
+         */
+        filecfg[1] = fileconf;
+        filecfg[2] = NULL;
+        strip = "tiered_storage=(shared=),";
+        WT_ERR(__wt_config_merge(session, filecfg, strip, &filestripped));
```

Comment:
These lines occur pretty similarly at least 3 or 4 times.  Maybe could be refactored into a function?

Reply:
Thanks. I tried to reduce the duplicate code wherever possible.